### ISSUE TITLE
Add "swagger" to the package description

### ISSUE THREAD

**Issue by schveiguy** [vnayar/openapi-client#3]
Someone was mentioning a swagger-based dub package, and when I searched for swagger, I got 0 results.

I believe if you put `swagger` in the description somewhere, the registry can find it while searching.

**Comment by vnayar** [comment]
Hmm, this is odd.  I was about to make this change, when I opened up the documentation here: https://code.dlang.org/packages/openapi-client

The first paragraph reads:
```
The purpose of this tool is to generate client libraries that are compatible with the REST API of a service, whose interface is described via an OpenAPI/Swagger specification document.

The format of OpenAPI specifications can be found here: https://swagger.io/specification/
```

I had assumed that the package registry considered the README, but it seems like it doesn't.  I'll add it to the description in the `dub.sdl`.

**Comment by vnayar** [comment]
Fixed in d39f6027b14d5f54f86b2c289a154c6566c7dd3b.
